portfolio/src/main/webapp/js/scripts.js

```diff
@@ -41,11 +41,29 @@
     });
 })(jQuery); // End of use strict
 
-/**
- * Function to fetch a message from server and convert it to text
- */
-async function getMessage() {
-  const response = await fetch('/data');
-  const message = await response.text();
-  document.getElementById('message-container').innerText = message;
+
+function loadComments() {
+    fetch('/list-comments')
+    .then(response => response.json())
+    .then((comments) => 
+    {
+        const commentListElement = document.getElementById('comment-list');
+        comments.forEach((comment) => 
+        {
+            commentListElement.appendChild(createCommentElement(comment));
```

Comment:
I like the code structure here and below! The code seems self explanatory (with well chosen variable/function names) and is easy to read.

Reply:
Thank you!